docs/java/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Java ♡ Ethereum
```

Comment:
Could we update this to "Ethereum for Java Developers"? Better for SEO & more descriptive for social share metadata.

Reply:
Fixed!